Mail/Components/MailboxListView.swift

```diff
@@ -32,17 +32,19 @@ struct MailboxListView: View {
             HStack(alignment: .center, spacing: 0) {
                 Text(MailResourcesStrings.Localizable.buttonAccountAssociatedEmailAddresses)
                     .textStyle(.bodySmallSecondary)
+                    .padding([.top, .bottom, .leading], value: .regular)
 
-                Spacer(minLength: UIPadding.regular)
+                Spacer(minLength: 0)
```

Comment:
Can you remove the Spacer? To do so, you have to add a .frame(maxWidth: .infinity) to the Text above.
We one read that a Spacer can lead to worse performance than a frame